apps/spark/src/main/java/com/linkedin/openhouse/jobs/scheduler/tasks/OperationTasksBuilder.java

```diff
@@ -249,6 +257,9 @@ public List<OperationTask<?>> buildOperationTaskList(
         return prepareDataLayoutOperationTaskList(jobType, properties, operationMode, otelEmitter);
       case ORPHAN_DIRECTORY_DELETION:
         return prepareTableDirectoryOperationTaskList(jobType, operationMode, otelEmitter);
+      case TABLE_DIRECTORY_CLEANUP:
```

Comment:
```suggestion
      case TABLE_DIRECTORY_DELETION:
```
To keep the name consistent with ORPHAN_FILES_DELETION and ORPHAN_DIRECTORY_DELETION